service/controller/resource/s3object/desired.go

```diff
@@ -2,17 +2,28 @@ package s3object
 
 import (
 	"context"
+	"fmt"
 	"sync"
 
-	"github.com/giantswarm/certs"
+	"github.com/giantswarm/apiextensions/pkg/apis/release/v1alpha1"
+	gscerts "github.com/giantswarm/certs"
+	"github.com/giantswarm/k8scloudconfig/v_4_9_1"
 	"github.com/giantswarm/microerror"
 	"github.com/giantswarm/randomkeys"
 	"golang.org/x/sync/errgroup"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
+	"github.com/giantswarm/aws-operator/pkg/label"
 	"github.com/giantswarm/aws-operator/service/controller/controllercontext"
+	"github.com/giantswarm/aws-operator/service/controller/internal/cloudconfig"
 	"github.com/giantswarm/aws-operator/service/controller/key"
 )
 
+const (
+	kubectlVersion              = "9ccdc9dc55a01b1fde2aea73901d0a699909c9cd" // 1.15.5
```

Comment:
This is a little ugly, but since I was moving all images out of k8scloudconfig and we're not storing this in the Release CR and we're not tagging `docker-kubectl` with kubernetes versions yet, we need to hardcode to the 1.15.5 image.

Reply:
I think we used the key package as a more central place for these kinds of magic strings.

Reply:
You're right. I was being lazy. I've moved this to `key` now.